### PR TITLE
KOA-5481 Update bar chart to semantic colours

### PR DESCRIPTION
**Pull request by marianeum**
Also took the chance to clean up some things that are no longer needed or don't match the current design

Updated to semantic colours and cleaned up some things that were not matching Figma designs.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/barchart/internal/ChartLineDecoration.kt [hunk 1]
```diff
@@ -48,26 +47,15 @@ internal class ChartLineDecoration(
     }
   }
 
-  override fun onDraw(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
```
Comment (marianeum) [inline Backpack/src/main/java/net/skyscanner/backpack/barchart/internal/ChartLineDecoration.kt:51]: we only want to draw the line over the content to match figma

File: app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt [hunk 2]
```diff
@@ -40,14 +40,15 @@ class BarChartStory : Story() {
...
+          selectedTitle = "Selected",
+          inactiveTitle = "No Price",
+          activeTitle = "Price",
         )
       )
     }
   }
 
-  private val random = Random()
+  private val random = Random(18735)
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt:51]: no more changing barchart screenshots!

Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt:51]: no more changing barchart screenshots!

Comment (bvitaliyg) [reply app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt:51]: But why 18735?

Comment (marianeum) [reply app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt:51]: Tried a few random numbers until I got one that had a nice mix of values

Comment (bvitaliyg) [reply app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt:51]: What about 42? It should answer anything

**Pull request merged**